scripts/irods/upgrade_configuration.py

```diff
@@ -332,9 +336,13 @@ def update_base(base, updates):
     # Add TLS properties for outbound traffic based on the service account client environment.
     add_verify_server_property = False
     if "irods_ssl_ca_certificate_file" in service_account_environment:
+        if "tls_client" not in new_server_config:
+            new_server_config["tls_client"] = {}
         new_server_config["tls_client"]["ca_certificate_file"] = service_account_environment.get("irods_ssl_ca_certificate_file", None)
```

Comment:
`.get(key, None)` can be replaced with `[key]` because of the if-statement.

Same for the code that follows.

Reply:
Done.